tool_quality_checks/importer.py

```diff
@@ -6,18 +6,83 @@
 import logging
 from osgeo import ogr
 from os.path import basename
+from OASDGLDatachecker.tool_quality_checks.correct_import_file import (
+    correct_vector_layer,
+)
 
 logger = logging.getLogger(__name__)
 ogr.UseExceptions()
 
 
-def copy2pg_database(settings, in_filepath, layer_name, schema="public"):
+def importer(db, settings):
+    """
+        Loads your input files into the database for checks
+    """
+    # check if relevant parameters are there:
+    if not hasattr(settings, "manhole_layer") or not hasattr(settings, "pipe_layer"):
+        logger.error("One of the input file path is missing, like manhole or pip")
+        raise AttributeError()
+
+    # initialize source schema
+    db.create_schema("src")
+
+    import_file_based_on_filetype(
+        db, settings, settings.manhole_layer, "putten_" + settings.import_type
+    )
+    import_file_based_on_filetype(
+        db, settings, settings.pipe_layer, "leidingen_" + settings.import_type
+    )
+
+    if settings.import_type == "gbi":
+        sql_relpath = os.path.join("sql", "sql_gbi_to_3di.sql")
+        sql_abspath = os.path.join(
+            os.path.abspath(os.path.dirname(__file__)), sql_relpath
+        )
+        db.execute_sql_file(sql_abspath)
+
+
+def import_file_based_on_filetype(db, settings, file_path, out_name):
+    """
+        Check file type and send request to copy to database function
+    """
+
+    if not os.path.isfile(file_path):
+        logger.error("File %s does not exists on your computer" % file_path)
```

Comment:
```suggestion
        logger.error("File %s does not exists " % file_path)
```

"On your computer" is niet echt nodig. En het kan zelfs verwarren als het script later op een server gedraaid wordt.